src/style.css

```diff
@@ -0,0 +1,97 @@
+@import "./styles/stars.css";
+@import "./styles/reset.css";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+
+  color: #060b35;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.description {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  color: #616070;
+  margin-bottom: 19px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__price-wrapper {
+  display: flex;
+  margin-bottom: 14px;
+  justify-content: space-between;
+  justify-items: center;
+}
+
+.card__price-word {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+
+}
+
+.card__button {
+  display: flex;
+  width: 100%;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  background: rgba(0, 172, 220, 1);
+  border-radius: 5px;
+}
+
+.card__button--hover:hover {
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
checklist p9

Reply:
I'm so sorry. My bad, you had to fix my mistakes. I hope my work is better now.